css/style.css

```diff
@@ -0,0 +1,226 @@
+@import url('https://fonts.googleapis.com/css?family=Lato:300,400,700&subset=latin-ext');
+
+*{
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+html{
+  font-size: 10px; /* 1rem = 10px */
+}
+body{
+  height: 100vh;
+  font-size: 1.6rem;
+  font-family: 'Lato', sans-serif;
+  background: linear-gradient(90deg, #83EB94 0%, #47D95E 100%);
+  /* background-attachment: fixed; */
+  /* background-repeat: no-repeat; */
+  background-size: contain;
+  color: #fff;
+}
+
+a{
+  text-decoration: none;
+}
+button{
+  font-family: roboto, sans-serif;
```

Comment:
Ustawiasz fonta, którego nie załadowałeś do projektu. Chciałeś chyba Lato wrzucić :)

Reply:
No tak, to copy/paste z resetu mojego jak zaczynam każdy projekt ;) Czcionkę przy okazji zmienia na bardziej przyjazną. Będę musiał pamiętać by to zmianiać.

Reply:
Sam sobie odpowiem, inherit lepiej się w tym przypadku sprawdzi:)